readme.md

```diff
@@ -0,0 +1,119 @@
+# MLPipeline 
+
+This is simple project which mocks a workflow of machine-learning pipeline. Project is implemented in python 3.9 although I guess anything 3.7+ should be fine.
+
+Project was created based on assignment in _assignment.md_ file. At beginning, I worked with structure of _.yaml_ as defined in assignment, however everything in the original yaml is list for no reason. For purposes of tests and ease of pipeline assembly I switched to slightly more normal implementation later. Still, there are residuals of original implementation - components are still lists, and whole thing is just begging to be OrderedDict. 
```

Comment:
I'm sorry, that was a mistake in the assignment. It's brand new and there are some glitches. However, I'm wondering whether reaching out about this issue wouldn't have been more constructive than implementing both versions and complaining about it in 6 places in the code/docs.

Reply:
Absolutely! However it was part of the assignment as well as two extra complex pipelines that you have insisted upon above so make up your mind :) No, seriously, it was late evening 2 days before deadline when I had some time to work on this homework and writing to Tomas expecting quick reply was not an option. So I did it, just to be sure that if it IS a part of the "TEST" than it would be mended. I thought that this is actually something that you get from some 3rd party library.